src/pages/home/index.js

```diff
@@ -3,11 +3,41 @@ import ContentTitle from "../../components/home/contentTiltle";
 import CategoryContainer from "../../components/home/categoryContainer";
 import ProductCard from "../../components/common/ProductCard";
 import topProductsData from "../../data/topProductsData";
+import Loader from "../../components/common/Loader";
+import { User } from "../../context/userContext";
+import axios from "axios";
+
 export const Home = () => {
+  const {state} = User()
+
+  const addToWishListHandler = async ({ item }) => {
+    try {
+      const config = {
+        headers: {
+          authorization: localStorage.getItem("token"),
+        },
+      };
+      const data = {
+        product: item,
+      };
+      await axios.post("/api/user/wishlist", data, config);
+      console.log("working from home")
+
```

Comment:
you can remove  this

Reply:
Made all the required changes in the branch - Cart